tools/import-mbox.py

```diff
@@ -327,7 +327,7 @@ def mailgen(_list):
                             }
                         },
                     )
-                    if res and res["hits"]["total"] > 0:
+                    if res and len(res["hits"]["total"]) > 0:
```

Comment:
Not sure the fix is correct: res["hits"]["total"] is a number, not an array.

What is the error message?